src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -398,17 +399,17 @@ class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchC
 
   val UNPARSABLE_CODE_PLACEMENT = "Unparsable code: "
 
-  private def toMichelsonScript[T <: MichelsonElement:Parser](json: Any): String = {
+  private def toMichelsonScript[T <: MichelsonElement](json: Any)(implicit parser: Parser[T], tag: ClassTag[T]): String = {
     Some(json).collect {
       case t: String => convert[T](t)
       case t: Micheline => convert[T](t.expression)
     } match {
       case Some(Right(value)) => value
       case Some(Left(t)) =>
-        logger.error(s"Error during converting Michelson format: $json", t)
+        logger.error(s"Error during converting Michelson format to (${tag.runtimeClass}): $json", t)
```

Comment:
Detail: I think the sentence is wrong, but may be mistaken

"Error while converting ..." or "Error during conversion of ..." should be correct?